apps/air-discount-scheme/backend/src/app/modules/nationalRegistry/test/unit/nationalRegistry.service.spec.ts

```diff
@@ -243,5 +245,67 @@ describe('NationalRegistryService', () => {
       expect(cacheManagerSetSpy).not.toHaveBeenCalled()
       expect(result).toEqual(children)
     })
+
+    it('should not fetch children of a child', async () => {
```

Comment:
This test doesn' really work because the `nationalRegistryFamilyLookupResponse` only have a single child and you are fetching relations for it. So maybe add another child to the response?